integration-libs/cdc/manage-users/manage-users-button/cdc-list.component.ts

```diff
@@ -0,0 +1,33 @@
+/*
```

Comment:
change the enclosing folder name from `manage-users-button` to `manage-users-list`

Reply:
done